### PR TITLE
Add note on Desktop installer size reduction

### PR DESCRIPTION
**Pull request by usha-mandya**
Added a note in Desktop 2.3.0.2 release notes about the reduction in the Desktop installer size.

### CODE REVIEW

**Comment by usha-mandya** [comment]
> do we need a mention that some components are now downloaded on demand?

Not sure. @stephen-turner Should we mention this?

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Docker Desktop introduces a new onboarding tutorial upon first startup. The Quic
 
 ### Bug fixes and minor changes
 
+- Reduced the size of the Docker Desktop installer from 710 MB to 445 MB.
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:34]: 708 -> 456

File: docker-for-windows/release-notes.md [hunk 2]
```diff
@@ -54,6 +54,7 @@ For information about Edge releases, see the [Edge release notes](edge-release-n
 
 **Other fixes**
 
+- Reduced the size of the Docker Desktop installer from 960 MB to 400 MB.
```
Comment (stephen-turner) [inline docker-for-windows/release-notes.md:57]: 960 -> 409

**Comment by stephen-turner** [comment]
> > do we need a mention that some components are now downloaded on demand?
> 
> Not sure. @stephen-turner Should we mention this?

I don't think we need to mention that.

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/release-notes.md]: Updated.

File: docker-for-windows/release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/release-notes.md]: Updated.

**Pull request merged**